### PR TITLE
dependencies update

### PR DESCRIPTION
**Pull request by debris**
updated deps and fixed tests which failed due to regression in serde codegen

### CODE REVIEW

**Comment by keorn** [comment]
Seems to work, but now I get the same error with `ethstore`, I guess it needs a similar pr:
```
   Compiling ethstore v0.1.0 ()                                   │
ethstore/build.rs:33:33: 33:46 error: mismatched types [E0308]                                       │
ethstore/build.rs:33         serde_codegen::register(&mut registry);                                 │
                                                     ^~~~~~~~~~~~~                                   │
ethstore/build.rs:33:33: 33:46 help: run `rustc --explain E0308` to see a detailed explanation       │
ethstore/build.rs:33:33: 33:46 note: expected type `&mut syntex::Registry`                           │
ethstore/build.rs:33:33: 33:46 note:    found type `&mut inner::syntex::Registry`                    │
error: aborting due to previous error  
```

**Comment by debris** [comment]
I'm on it

File: src/spec/operation.rs [hunk 1]
```diff
@@ -19,20 +19,17 @@ pub enum Operation {
 impl Deserialize for Operation {
 	fn deserialize<D>(deserializer: &mut D) -> Result<Self, D::Error>
 		where D: Deserializer {
-		let v = try!(Value::deserialize(deserializer));
-		if let Value::Object(ref map) = v.clone() {
-			if let Some(&Value::String(ref s) = map.get("type") {
```
Comment (oli-obk) [inline src/spec/operation.rs]: how did that ever work? you were missing a bracket

Comment (debris) [inline src/spec/operation.rs]: you are right! I have no idea! :)

**Pull request merged**